utils/oscap-xccdf.c

```diff
@@ -446,6 +446,13 @@ static void report_missing_profile(const struct oscap_action *action)
 		"$ oscap info \"%s\"\n", action->profile, action->f_xccdf);
 }
 
+static void report_multiple_profile_matches(const struct oscap_action *action)
+{
+	fprintf(stderr,
+			"At least two profiles matched the given suffix. Use a more specific suffix "
+			"to get an exact match.\n");
```

Comment:
Hint at oscap info here, similar to what's done in report_missing_profile.
Also please fix the indents to be same as report_missing_profile, not sure why it's indented so much.